src/python/director/ikconstraintencoder.py

```diff
@@ -43,25 +38,3 @@ def getPlanPoses(constraints, ikPlanner):
     poses = sorted([c.postureName for c in constraints if hasattr(c, 'postureName')])
     poses = {poseName:list(ikPlanner.jointController.getPose(poseName)) for poseName in poses}
     return poses
-
-
-class IKConstraintEncoder(object):
-    def __init__(self,ikPlanner):
-        self.ikPlanner = ikPlanner
-
-    def publishConstraints(self,constraints,messageName='PLANNER_REQUEST'):
```

Comment:
Aside from the above issue, it seems this is quite minimal renaming.

That said I haven't actually tested it